js/input.js

```diff
@@ -0,0 +1,23 @@
+var subscribe_button = document.querySelector(".subscribe_button");

```

Comment:
use const/let , stop using var is the old way of doing things, and can lead in big problems in the project since a var can be reaasigned and also i does take more memory then a const/let as far as i know, because of the hoisting